### PR TITLE
[USETUP] Home/End keys in Boot Loader Page

### PR DESCRIPTION
**Pull request by feel-the-dz3n**
## Purpose

Allow to jump to the first or the last entry in bootloader installation page list using Home/End keys.
gif

### CODE REVIEW

File: base/setup/usetup/usetup.c [hunk 1]
```diff
@@ -4019,6 +4019,24 @@ BootLoaderPage(PINPUT_RECORD Ir)
 
             CONSOLE_InvertTextXY(8, Line, 60, 1);
         }
+        else if ((Ir->Event.KeyEvent.uChar.AsciiChar == 0x00) &&
+            (Ir->Event.KeyEvent.wVirtualKeyCode == VK_HOME))  /* HOME */
```
Comment (HBelusca) [inline base/setup/usetup/usetup.c]: Indentation fix please.

File: base/setup/usetup/usetup.c [hunk 2]
```diff
@@ -4019,6 +4019,24 @@ BootLoaderPage(PINPUT_RECORD Ir)
...
+        {
+            CONSOLE_NormalTextXY(8, Line, 60, 1);
+
+            Line = 12;
+
+            CONSOLE_InvertTextXY(8, Line, 60, 1);
+        }
+        else if ((Ir->Event.KeyEvent.uChar.AsciiChar == 0x00) &&
+            (Ir->Event.KeyEvent.wVirtualKeyCode == VK_END))  /* END */
```
Comment (HBelusca) [inline base/setup/usetup/usetup.c]: Indentation fix please.

File: base/setup/usetup/usetup.c (hunk 1 above)
Comment (feel-the-dz3n) [reply base/setup/usetup/usetup.c]: Done.

File: base/setup/usetup/usetup.c (hunk 2 above)
Comment (feel-the-dz3n) [reply base/setup/usetup/usetup.c]: Done.

**Pull request merged**